lib/matplotlib/_constrained_layout.py

```diff
@@ -291,21 +291,24 @@ def _make_margin_suptitles(fig, renderer, *, w_pad=0, h_pad=0):
 
     if fig._suptitle is not None and fig._suptitle.get_in_layout():
         p = fig._suptitle.get_position()
-        fig._suptitle.set_position((p[0], 1 - h_pad_local))
-        bbox = inv_trans_fig(fig._suptitle.get_tightbbox(renderer))
-        fig._layoutgrid.edit_margin_min('top', bbox.height + 2.0 * h_pad)
+        if fig._suptitle._autopos:  # default position, so assume we can move
```

Comment:
as these are just normal `Text` objects, maybe we should do `getattr(.., '_auto_pos', False)` in case the users have done something funny here and assigned their own `Text` object into place?